### DISCUSSION TITLE
4-Pack Steam Option

### DISCUSSION THREAD

**Discussion by ThePiggyPlane** [FakeFishGames/Barotrauma#5275] [Ideas and suggestions]
I _love_ Barotrauma, so much so that I'm fairly addicted to gifting the game to friends. However, this led me to thinking, why not add a 4-pack sale option to the Steam store listing, to gift copies of the games to friends? In my opinion, it would definitely make buying the game for friends a lot more convenient, and may attract groups of friends looking for a multiplayer experience.

**Comment by Anikyte** [comment]
IIRC, Steam doesn't generally allow games to sell 4-packs anymore.
